services/movie.py

```diff
@@ -1,11 +1,12 @@
+from django.db import transaction
 from django.db.models import QuerySet
-
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Consider using `Optional[list[int]]` from the `typing` module for `genres_ids` and `actors_ids` to explicitly indicate that these parameters can be `None`.